### DISCUSSION TITLE
Documentation - should it read Menu::make ?

### DISCUSSION THREAD

**Discussion by HeartlandTechie** [orchidsoftware/platform#1690] [Q&A]
From the starter example:

 `  ItemMenu::label('Email sender')
            ->icon('envelope-letter')
            ->route('platform.email')
            ->title('Tools')`

Mine blew up . . . I switched it to this:

`      Menu::make('Email sender')
                ->icon('envelope-letter')
                ->route('platform.email')
                ->title('Tools'),`

**Comment by tabuna** [comment]
Hi @HeartlandTechie. Thanks for pointing this out.
 We recently released version 10, where we changed the syntax. I fixed this point in the documentation.

**Comment by wsdnathan** [comment]
Not just ItemMenu to Menu but also `use Orchid\Screen\Actions\Menu;`
